ccd-definition/ComplexTypes/1_JudgeAndLegalAdvisor.json

```diff
@@ -70,5 +70,14 @@
     "ElementLabel": "Justices' Legal Adviser's full name",
     "FieldType": "Text",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "JudgeAndLegalAdvisor",
+    "ListElementCode": "judgeEmailAddress",
+    "FieldType": "Email",
+    "ElementLabel": "Email Address",
+    "FieldShowCondition": "judgeTitle!=\"\"",
```

Comment:
Is this needed? Should this not be shown at all times?

Reply:
When you first land on the screen for the 1st time none of the radio buttons are selected. The name & email fields are hidden. Upon selection these fields are shown.